src/test/java/com/metamx/metrics/SigarLoadTest.java

```diff
@@ -0,0 +1,18 @@
+package com.metamx.metrics;
+
+import com.metamx.emitter.service.ServiceEmitter;
+import java.lang.management.RuntimeMXBean;
+import junit.framework.Assert;
+import org.hyperic.sigar.Sigar;
+import org.hyperic.sigar.SigarException;
+import org.junit.Test;
+
+public class SigarLoadTest
+{
+  @Test
+  public void testSigarLoad() throws SigarException
+  {
+    Sigar sigar = SigarUtil.getSigar();
+    Assert.assertEquals(sigar.getCpuInfoList().length, Runtime.getRuntime().availableProcessors());
```

Comment:
we can just test that it's not empty, assuming the user running the test always has the permissions to get that information, which may not always be true.

Maybe a better test is to find something that any user will always have access to, regardless of permissions and test that?